pkg/reco/workflow.go

```diff
@@ -91,6 +91,7 @@ func (rw *RecommendationWorkflowImpl) Execute(ctx context.Context, wm WorkloadMe
 		rw.logger.Error(err, "Error while generating recommendation")
 		return nil, nil, nil, errors.New("Unable to generate recommendation")
 	}
+	targetRecoConfig = transformTargetRecoConfig(targetRecoConfig)
```

Comment:
Add a comment as a reminder to log an event/metric about the recommendation config generated by the recommender.